packages/kbn-securitysolution-alerts-grouping/src/components/grouped_tables.tsx

```diff
@@ -14,19 +15,17 @@ import {
 } from '@elastic/eui';
 import type { Filter } from '@kbn/es-query';
 import React, { useMemo, useState } from 'react';
-import { firstNonNullValue } from '../../../../../common/endpoint/models/ecs_safety_helpers';
-import { createGroupFilter } from '../accordion_panel/helpers';
-import { tableDefaults } from '../../../store/data_table/defaults';
-import { defaultUnit } from '../../toolbar/unit';
-import type { BadgeMetric, CustomMetric } from '../accordion_panel';
-import { GroupPanel } from '../accordion_panel';
-import { GroupStats } from '../accordion_panel/group_stats';
-import { EmptyGroupingComponent } from '../empty_resuls_panel';
-import { GroupingStyledContainer, GroupsUnitCount } from '../styles';
-import { GROUPS_UNIT } from '../translations';
-import type { GroupingTableAggregation, RawBucket } from '../types';
+import { defaultUnit, firstNonNullValue } from '../helpers';
+import { createGroupFilter } from './accordion_panel/helpers';
+import type { BadgeMetric, CustomMetric } from './accordion_panel';
+import { GroupPanel } from './accordion_panel';
+import { GroupStats } from './accordion_panel/group_stats';
+import { EmptyGroupingComponent } from './empty_resuls_panel';
+import { GroupingStyledContainer, GroupsUnitCount } from './styles';
+import { GROUPS_UNIT } from './translations';
+import type { GroupingTableAggregation, RawBucket } from './types';
 
-export interface GroupingContainerProps {
+export interface GroupedTablesProps {
   badgeMetricStats?: (fieldBucket: RawBucket) => BadgeMetric[];
   customMetricStats?: (fieldBucket: RawBucket) => CustomMetric[];
   data: GroupingTableAggregation &
```

Comment:
Lets replace it with `data?: GroupingTableAggregation & GroupingFieldTotalAggregation;`